File: Desktop/kroz/c++/Laith/challenge1.cpp
#include <iostream>

using namespace std;

int main()
{
    int a;
    cout << "Type the 1st number : ";
    cin >> a;
    int b;
    cout << "Type the 2nd number : ";
    cin >> b;
    int res;
    cout << "Addition '+'" << endl;
    cout << "Subtraction '-'" << endl;
    cout << "Product '*' " << endl;
    cout << "Division '/'" << endl;
    char choice;
    cout << "Choose an operation : ";
    cin >> choice;
    if ( choice == '+'){
        res = a + b;
    } else if( choice == '-'){
        res = a - b;
    } else if( choice == '*'){
        res = a * b;
    } else {
        res = a / b;
    }
    cout << "The result = " << res << endl;


}

File: Desktop/kroz/c++/Laith/challenge1.py
a=int(input("Type the 1st number"))
b=int(input("Type the 2nd number"))
print("Addition '+'")
print("Subtraction '-'")
print("Product '*'")
print("Division '/'")
choice=input("Choose an operation : ")
while ((choice != '+') and ( choice != '-') and ( choice != '*') and ( choice != '/')):
    print("Invalid Input")
    choice=input("Choose an operation : ")
if ( choice == '+'):
    print("The result = ", a+b)
elif ( choice == '-'):
    print("The result = ", a-b)
elif ( choice == '*'):
    print("The result = ", a*b)
else :
    print("The result = ", a/b)
    


